README.md

```diff
@@ -588,6 +588,7 @@ _CMS are a practical way to setup a website with many features. CMS often come w
 - [Saleor](http://getsaleor.com/) - Django based open-sourced e-commerce storefront. ([Demo](https://demo.getsaleor.com/), [Source Code](https://github.com/mirumee/saleor)) `BSD-3-Clause` `Python`
 - [Shuup](https://www.shuup.com/) - Django powered fully customizable open source e-commerce framework for small and large sites. ([Source Code](https://github.com/shuup/shuup)) `AGPL-3.0` `Python`
 - [Shopware Community Edition](https://shopware.com/community/) - PHP based open source e-commerce software made in Germany. ([Demo](https://www.shopware.com/en/test-demo/), [Source Code](https://github.com/shopware/platform)) `MIT` `PHP`
+- [Solidus](https://solidus.io/) - A free, open-source ecommerce platform that gives you complete control over your store. ([Demo](http://solidemo.herokuapp.com/), [Source Code](https://github.com/solidusio/solidus)) `BSD-3-Clause` `Ruby`
```

Comment:
Hey @matteocellucci, can you please use http://demo.solidus.io/ ? We are switching to this new URL everywhere, thanks!

Reply:
Resolved